src/crypto/crypto.ts

```diff
@@ -13,6 +13,7 @@ import { ensure, Nullable, Optional } from '@salesforce/ts-types';
 import { AsyncOptionalCreatable, env } from '@salesforce/kit';
 import { Logger } from '../logger';
 import { Messages } from '../messages';
+import { Cache } from '../util/cache';
```

Comment:
checking my understanding: By putting the cache inside Crypto, each consumer of Crypto (typically configfile or one of its descendants), the cache is scoped to that instance of the configFile(etc) class?

Reply:
@mshanemc No Cache is a singleton, so same instance is used for all references across the app.